its/pom.xml

```diff
@@ -19,6 +19,7 @@
 
   <properties>
     <jetty.version>9.4.41.v20210516</jetty.version>
+    <maven.compiler.release>8</maven.compiler.release>
```

Comment:
why is this necessary?

Reply:
I think we can target Java 11 , it's internal and we should be on the latest LTS

Or is it about the VM that doesn't have it yet?

Reply:
It was failing with an error about the scanner so I updated it, I planned to do the migration to the new machines and fix it at that point, however it might not even be related to JDK availability as according to this page JAVA 11 is available on the current machines: https://github.com/actions/virtual-environments/blob/main/images/win/Windows2016-Readme.md#java

Reply:
See email `New Year update` from Julien Carsique and reply from Julien Henry. An explicit configuration is needed because of some new parent POM checks.

(While I've also suggested v11, I do not contribute to discussion 8 vs 11 here)